Atomic/util.py

```diff
@@ -36,8 +36,8 @@
 ATOMIC_INSTALL_JSON = os.environ.get('ATOMIC_INSTALL_JSON', os.path.join(ATOMIC_VAR_LIB, 'install.json'))
 
 GOMTREE_PATH = os.environ.get("GOMTREE_PATH", "/usr/bin/gomtree")
-BWRAP_OCI_PATH = os.environ.get("BWRAP_OCI", "/usr/bin/bwrap-oci")
-RUNC_PATH = os.environ.get("RUNC", "/bin/runc")
+BWRAP_OCI_PATH = os.environ.get("BWRAP_OCI_PATH", "/usr/bin/bwrap-oci")
+RUNC_PATH = os.environ.get("RUNC_PATH", "/usr/bin/runc")
```

Comment:
I think ubuntu's runc ends up in /usr/sbin now ... do you care?

Reply:
yes, by default it doesn't work, but we can't make happy both Fedora and Ubuntu :-( so I picked Fedora and expect RUNC_PATH to be specified on Ubuntu